src/storm/values.yaml

```diff
@@ -30,10 +31,15 @@ supervisor:
   service:
     name: supervisor
     port: 6700
+  childopts: "-Xmx256m -ea"
   resources: {}
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  worker:
+    heap:
+      memory:
+        mb: 768
```

Comment:
question: what about renaming this to `heap_memory_mb`?

Reply:
cleaner, like it 👍 
at first, i wanted to re-use the same names as original ⚡